### PR TITLE
Fix AI teleporting tickets to its core

### PR DESCRIPTION
**Pull request by Nazsgull**
## What Does This PR Do
Fixes AI interacting with the ticket machine.

## Why It's Good For The Game
AI will no longer teleport tickets to its core when interacting with the ticket machine.

## Changelog
:cl: Nazsgull
del: AI can no longer interact with ticket machine.
fix: AI can no longer teleport tickets to its core. 
/:cl:

### CODE REVIEW

**Comment by Ty-Omaha** [comment]
Here I am thinking there was some sort of real AI like a neural network or something.

File: code/modules/paperwork/ticketmachine.dm [hunk 1]
```diff
@@ -167,6 +167,8 @@
 		user.adjust_fire_stacks(1)
 		user.IgniteMob()
 
+/obj/machinery/ticket_machine/attack_ai(mob/user)//AI cannot interact with ticket_machine
+	return
 /obj/item/ticket_machine_ticket
```
Comment (SteelSlayer) [inline code/modules/paperwork/ticketmachine.dm:172]: Usually you want to put any procs *after* the object definition, so something like: 
```js
/obj/item/ticket_machine_ticket
    name = "Ticket"
    ... // other stuff here, description, etc.

/obj/machinery/ticket_machine/attack_ai(mob/user)//AI cannot interact with ticket_machine
	return
```

It just looks better that way. Also it's good to have a blank line between every proc or path

Comment (Fox-McCloud) [reply code/modules/paperwork/ticketmachine.dm:172]: Yep, for sure.

File: code/modules/paperwork/ticketmachine.dm [hunk 2]
```diff
@@ -167,9 +167,11 @@
...
 	name = "Ticket"
-	desc = "A ticket which shows your place in the Head of Personnel's line. Made from Nanotrasen patented NanoPaper�. Though solid, its form seems to shimmer slightly. Feels (and burns) just like the real thing."
+	desc = "A ticket which shows your place in the Head of Personnel's line. Made from Nanotrasen patented NanoPaper®. Though solid, its form seems to shimmer slightly. Feels (and burns) just like the real thing."
```
Comment (Fox-McCloud) [inline code/modules/paperwork/ticketmachine.dm:174]: What changed here?

Comment (Nazsgull) [reply code/modules/paperwork/ticketmachine.dm:174]: Encoding got changed, I changed it back but I had removed the corrupted symbol, so I put it back.

File: code/modules/paperwork/ticketmachine.dm (hunk 1 above)
Comment (Nazsgull) [reply code/modules/paperwork/ticketmachine.dm:172]: Yeah but the method I added is for the machine, so it goes after the other machine methods, not near the ticket ones.

Comment (SteelSlayer) [reply code/modules/paperwork/ticketmachine.dm:172]: You're right, I mixed up the two similar paths here. So the way you have it is fine. The blank line after the return would still be nice though.

Comment (Fox-McCloud) [reply code/modules/paperwork/ticketmachine.dm:172]: still has spacing issues.

File: code/modules/paperwork/ticketmachine.dm (hunk 2 above)
Comment (Fox-McCloud) [reply code/modules/paperwork/ticketmachine.dm:174]: I'd prefer this not show up at all.

**Pull request closed without merging**